commons-geometry-examples/examples-io/src/test/java/org/apache/commons/geometry/examples/io/threed/obj/OBJWriterTest.java

```diff
@@ -65,7 +65,6 @@ public void testClose_calledMultipleTimes() throws IOException {
         // act/assert
         try (OBJWriter meshWriter = new OBJWriter(writer)) {
             meshWriter.close();
-            meshWriter.close();
```

Comment:
@darkma773r is this a bug or intentionally calling `close()` twice?

Reply:
Also, what is the need to call `close()` for _try-with-resources_?

Reply:
No idea.

Reply:
Twice is fine. I was just making _extra_ sure that we hit that `if` condition in `close()` :-)

Reply:
:)